tests/DoctrineTest/Doctrine_UnitTestCase.php

```diff
@@ -48,6 +48,7 @@ class Doctrine_UnitTestCase extends UnitTestCase
     protected $generic = false;
     protected $conn;
     protected $adapter;
+    /** @var Doctrine_Export */
```

Comment:
Can we not make this
```suggestion
    protected Doctrine_Export $export'
```
or will that cause us issues next time we try to upgrade?

Reply:
Kinda the same as the above really. We can do whatever we want with this fork, but the more it drifts away from the upstream, the more complicated future upgrades might be.